server/src/main/java/org/elasticsearch/TransportVersion.java

```diff
@@ -135,7 +136,7 @@ private static TransportVersion registerTransportVersion(int id, String uniqueId
      * Reference to the most recent transport version.
      * This should be the transport version with the highest id.
      */
-    public static final TransportVersion CURRENT = V_8_500_007;
+    public static final TransportVersion CURRENT = V_8_500_999;
```

Comment:
I will update this once the PR is ready, but did not want to contend with other changes as this may need multiple rounds of review.